lms/djangoapps/certificates/admin.py

```diff
@@ -96,9 +97,23 @@ class CertificateGenerationCommandConfigurationAdmin(ConfigurationModelAdmin):
     pass
 
 
+class CertificateDateOverrideAdmin(admin.ModelAdmin):
+    """
+    # Django admin customizations for CertificateDateOverride model
+    """
+    list_display = ('generated_certificate', 'date', 'reason', 'overridden_by')
```

Comment:
Don't we want `overridden_by` to be a read only field since we're autopopulating it?